schemas/user.py

```
@@ -69,6 +69,11 @@ class UserLoginResponse(BaseModel):
     token_type: str = "bearer"
 
 
+class TokenRefreshResponse(BaseModel):
+    access_token: str
+    token_type: str = "bearer"
```

Ruff (0.14.13)

Possible hardcoded password assigned to: "token_type"

(S105)